src/lib/ibcclient.ts

```diff
@@ -663,6 +692,7 @@ export class IbcClient {
     myClientId: string,
     proof: ConnectionHandshakeProof
   ): Promise<CreateConnectionResult> {
+    this.logger.info(`Connection open try for my client ${myClientId}`);
```

Comment:
Listing at least (1) counterparty clientId and maybe (2) counterparty connection id would be nice.
Or maybe we log both connection IDs shortly before return? (We parse the newly defined one there)